shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -725,23 +725,15 @@ - (void)keyboardWillChangeFrame:(NSNotification*)notification {
   CGFloat scale = [UIScreen mainScreen].scale;
 
   // The keyboard is treated as an inset since we want to effectively reduce the window size by the
-  // keyboard height. We also eliminate any bottom safe-area padding since they keyboard 'consumes'
-  // the home indicator widget.
+  // keyboard height. The Dart side will compute a value accounting for the keyboard consuming
```

Comment:
uber-nit: `keyboard-consuming` should be a hyphenated adjective. Unless I'm US-Englishing wrong :)

Reply:
Done